src/Cms/App.php

```diff
@@ -1187,7 +1187,15 @@ public function smartypants(string $text = null): string
         if ($options === true) {
             $options = [];
         }
-
+        
+        if ($this->multilang() === true) {
+            $languageSmartypants = $this->language()->smartypants() ?? [];
+        
+            if (empty($languageSmartypants) === false) {
+                $options = array_merge($options, $languageSmartypants);
```

Comment:
This operation will fail if the `smartypants` config option is set to `false` (you can't merge a boolean with an array).

Now that I think about it: What happens currently if the `$kirby->smartypants()` method is called with SmartyPants disabled in the config? Probably breaks somewhere else anyway. Maybe we should change line `1187` to check for `is_array($options) !== true` instead to make sure that `$options` is always an array, even if SmartyPants is disabled in the config.

Reply:
You was right! An error occurs when `smartypants` option set to `false` while `smartypants()` called.

> Argument 3 passed to Kirby\Cms\App::{closure}() must be of the type array, boolean given, called in ...\kirby\src\Cms\App.php on line 1191